source/capacity_settlement/tests/scenario_tests/wip_simple_effektbetaling/when/job_parameters.yml

```
@@ -0,0 +1,3 @@
+- orchestration_instance_id: 00000000-orch-id00-0000-000000000001
+- calculation_period_start: 2025-12-31 23:00:00
```

In wholesale we use "calculation_period_start_datetime" and "period_end_datetime"
```suggestion
- calculation_period_start_datetime: 2025-12-31 23:00:00
```